content/student/finance_questions.md

```diff
@@ -0,0 +1,21 @@
+title: К финансовым вопросам
+slug: finance_questions
```

Comment:
`slug` name should not contain `_`. `slug` should be all lower case, and match `[a-z0-9-]*`.